sdk/communication/communication-network-traversal/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-02-22-preview1)
+
+### Added
+
+- Added CommunicationRelayClient in preview.
```

Comment:
I think this can be more descriptive if this is what you want to release the preview with.
Take a look at the changelog.md in chat or sms for examples